docs/usage/cli.md

```diff
@@ -0,0 +1,136 @@
+
```

Comment:
If this file is dynamically generated it should not be committed and `.gitignore`ed instead. Otherwise, it might create unnecessary diffs on other PRs.

Reply:
@3xtr4t3rr3str14l bump

Reply:
done!